### PR TITLE
WINDUP-1011 Make the stats files human readable

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

File: utils/src/main/java/org/jboss/windup/util/ExecutionStatistics.java [hunk 1]
```diff
@@ -114,13 +115,16 @@ public void serializeTimingData(Path outputPath)
 
         try (FileWriter fw = new FileWriter(outputPath.toFile()))
         {
-            fw.write("Type~Number Of Executions~Total Milliseconds~Milliseconds per execution\n");
+            fw.write("Number Of Executions | Total Milliseconds | Milliseconds per execution | Type\n");
```
Comment (jsight) [inline utils/src/main/java/org/jboss/windup/util/ExecutionStatistics.java]: Doesn't this effectively make the delimiter be the "|" character?

File: utils/src/main/java/org/jboss/windup/util/ExecutionStatistics.java [hunk 2]
```diff
@@ -114,13 +115,16 @@ public void serializeTimingData(Path outputPath)
...
             for (Map.Entry<String, TimingData> timing : executionInfo.entrySet())
             {
                 TimingData data = timing.getValue();
                 long totalMillis = (data.totalNanos / 1000000);
                 double millisPerExecution = (double) totalMillis / (double) data.numberOfExecutions;
-                fw.write(timing.getKey() + "~" + data.numberOfExecutions + "~" + totalMillis + "~" + millisPerExecution);
+                fw.write(String.format("%6d, %6d, %8.2f, %s",
```
Comment (jsight) [inline utils/src/main/java/org/jboss/windup/util/ExecutionStatistics.java]: But down here, the delimiter appears to be ",".

File: reporting/impl/src/main/java/org/jboss/windup/reporting/rules/rendering/ExecutionTimeReportRuleProvider.java [hunk 3]
```diff
@@ -85,13 +87,11 @@ public void perform(GraphRewrite event, EvaluationContext context)
                     fw.write("Rule execution timings:\n\n");
                     for (RuleProviderExecutionStatisticsModel model : ruleProviderStatModels)
                     {
+                        fw.write(StringUtils.leftPad(String.valueOf(model.getTimeTaken() / 1000), 5));
```
Comment (jsight) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/rules/rendering/ExecutionTimeReportRuleProvider.java]: Is there a reason not to use String.format here?

Comment (OndraZizka) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/rules/rendering/ExecutionTimeReportRuleProvider.java]: No :) I'll change.

File: utils/src/main/java/org/jboss/windup/util/ExecutionStatistics.java (hunk 2 above)
Comment (OndraZizka) [inline utils/src/main/java/org/jboss/windup/util/ExecutionStatistics.java]: Right... hasty push. Fixing.

**Comment by jsight** [comment]
Have you verified what impact this will have on the performance scripts on wonka?

**Comment by OndraZizka** [comment]
No I haven't, I have mentioned at the meeting that I haven't found anything depending on it. I might have overlooked. Do you have some tip?

**Comment by OndraZizka** [comment]
Namely, I see /opt/perftest/Main.groovy but there's no such path at wonka.

**Comment by jsight** [comment]
@OndraZizka 

Looks like it exists on jenkins:
$ ssh jsightle@wonka.mw.lab.eng.bos.redhat.com 
$ ssh cloud-user@192.168.122.52
Last login: Mon May  2 11:57:05 2016 from gateway
$ cd /opt/perftest/
[cloud-user@jsightle-windup-jenkins perftest]$

**Comment by OndraZizka** [comment]
Note for myself: https://github.com/windup/windup-ci/blob/master/perf-test-jenkins/Main.groovy#L172

**Comment by OndraZizka** [comment]
Here's the PR for CI. I'm not sure how to test it, if someone has the env set up already, it would be nice to try. Otherwise I'll get to this later. 
https://github.com/windup/windup-ci/pull/3

**Comment by jsight** [comment]
It is easiest to test it on the server itself. I assume this means that you haven't tested the groovy changes?

**Comment by OndraZizka** [comment]
Nope, I haven't tested. I'll get back to mavenization, this is just something I wanted to have as I was looking if mavenization doesn't take too long.
Could you perhaps try test it please? Seems like you're better with Jenkins than me

**Pull request merged**